scripts/2_match_households_and_individuals.py

```diff
@@ -924,6 +953,19 @@ def get_interim_path(
             show_progress=True,
         )
 
+        # match remaining individuals
+        remaining_ids = spc_edited.loc[
+            ~spc_edited.index.isin(matches_ind.keys()), "id"
+        ].to_list()
+        matches_remaining_ind = match_remaining_individuals(
+            df1=spc_edited,
+            df2=nts_individuals,
+            matching_columns=["age_group", "sex"],
```

Comment:
Could update the matching_columns here to enable more precision when not using households: e.g. for employment status and urban rural classification.